cmd/feedback/feedback.go

```diff
@@ -80,17 +82,49 @@ type SurveyConfigInterface interface {
 // SurveyStore stores all available surveys.
 // New surveys should be added here.
 var SurveyStore = map[string]SlackSurvey{
-	// PlatformSurvey asks for general developer experience feedback
-	PlatformSurvey: {
-		Name:              PlatformSurvey,
-		PromptDisplayText: "Help make the Slack platform better",
-		PromptDescription: "Tell us about your experience as a Slack developer",
+	// SlackCLIFeedback asks for Slack CLI feedback using GitHub Issues
+	SlackCLIFeedback: {
+		Name:              SlackCLIFeedback,
+		PromptDisplayText: "Slack CLI",
+		PromptDescription: "Questions, issues, and feature requests about the Slack CLI",
+		SkipQueryParams:   true,
+		URL: url.URL{
+			RawPath: "https://github.com/slackapi/slack-cli/issues",
+		},
+		Info: func(ctx context.Context, clients *shared.ClientFactory) {
+			clients.IO.PrintInfo(ctx, false, fmt.Sprintf(
+				"%s\n%s\n",
+				style.Secondary("Ask questions, submit issues, or suggest features for the SLack CLI:"),
+				style.Secondary(style.Highlight("https://github.com/slackapi/slack-cli/issues")),
+			))
+		},
+		Trace: slacktrace.FeedbackMessage,
```

Comment:
note: In the future, we could make the trace optional or removed entirely. I think it's better to have 1 trace for all feedback options (`slacktrace.FeedbackMessage`) then include the selected feedback name as trace data.

Reply:
Heh I missed this in a quick review earlier! But I so agree! Thanks for opening #68 🙏 ✨